enzyme/Enzyme/Utils.cpp

```diff
@@ -2038,3 +2183,110 @@ llvm::FastMathFlags getFast() {
     f.set();
   return f;
 }
+
+void addValueToCache(llvm::Value *arg, bool cache_arg, llvm::Type *ty,
+                     llvm::SmallVector<llvm::Value *, 2> &cacheValues,
+                     llvm::IRBuilder<> &BuilderZ, llvm::Twine name) {
```

Comment:
`Twine` should always be passed as a const reference

```suggestion
void addValueToCache(llvm::Value *arg, bool cache_arg, llvm::Type *ty,
                     llvm::SmallVector<llvm::Value *, 2> &cacheValues,
                     llvm::IRBuilder<> &BuilderZ, const llvm::Twine &name) {
```